File: AdoptPet_Server/AdoptPet.Application/DTOs/Pet/PetDto.cs


namespace AdoptPet.Application.DTOs.Pet
{
    public class PetDto
    {
        public string PetName { get; set; } = string.Empty;
        public string PetDescription { get; set; } = string.Empty;
        public decimal PetWeight { get; set; }
        public string PetAge { get; set; } = string.Empty;
        public bool PetGender { get; set; }
        public byte PetDesexed { get; set; }
        public byte PetWormed { get; set; }
        public byte PetVaccined { get; set; }
        public byte PetMicrochipped { get; set; }
        public DateTime PetEntryDate { get; set; }
    }
}

File: AdoptPet_Client/src/app/shared/models/pet/PetDto.ts
import { Breed } from "../breed/breed";
import { Color } from "../color/Color";
import { PetBreed } from "./PetBreed";
import { PetColour } from "./PetColour";
import { PetImage } from "./PetImage";


export interface PetDto {
	id: number;
	petName: string;
	petDescription: string;
	petWeight: number;
	petAge: number;
	petGender: number;
	petDesexed: number;
	petWormed: number;
	petVaccined: number;
	petMicrochipped: number;
	petEntryDate: Date;
	status: number;
	isDeleted: number;

	// khoa ngoai
	volunteerId: number;
	ownerId: number;

	breed: Breed;
	colour: Color;	

	petImages: PetImage[];
}
